system-parachains/constants/src/lib.rs

```diff
@@ -57,3 +57,42 @@ pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 
 /// Treasury pallet id of the local chain, used to convert into AccountId
 pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"py/trsry");
+
+/// Parameters supporting async backing functionality.
+///
+/// Once all system chains have migrated to the new async backing mechanism, the parameters
+/// in this namespace will replace those currently defined in `super::*`.
+pub mod async_backing {
+	use frame_support::weights::{constants::WEIGHT_REF_TIME_PER_SECOND, Weight};
+	pub use parachains_common::BlockNumber;
+	use sp_runtime::Perbill;
+
+	/// This determines the average expected block time that we are targeting.
+	///
+	/// Blocks will be produced at a minimum duration defined by `SLOT_DURATION`. `SLOT_DURATION` is
+	/// picked up by `pallet_timestamp` which is in turn picked up by `pallet_aura` to implement `fn
+	/// slot_duration()`.
+	///
+	/// Change this to adjust the block time.
+	pub const MILLISECS_PER_BLOCK: u64 = 6_000;
+	pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
```

Comment:
We should keep this at 12s. Reduces stress once we enable elastic scaling. (Less frequent handovers)

Reply:
You mean only `SLOT_DURATION`? We keep 6s for `MILLISECS_PER_BLOCK`? Should it also be readjusted for People Chain? Now its set to 6s.

Reply:
> We should keep this at 12s. Reduces stress once we enable elastic scaling. (Less frequent handovers)

I'm for going down to 6s. If one guy being down, we only loose 6s and not 12s.

Reply:
We can do so later, if we think it is worthwhile. Going up again is less straight forward.

Risk wise it also seems more likely that we run into a situation where a longer slot duration is beneficial than the other way round.

Reply:
Do we have any data on how frequent collators go down in practice?

Reply:
> Going up again is less straight forward.

Going up is a straight forward as going down. (yeah right now requires a small migration, but if we really want the flexibility we can fix it in `polkadot-sdk`.

Other chains like Solana change the leader aka block author every 1.6 seconds :)

Reply:
We also had a discussion about censorship resistance yesterday with Web3. This is a problem right now and should be fixed especially for crucial chains like asset hub. While longer slot times don't directly fix the issue, shorter ones certainly don't help. I would keep it as is, until we have a solution that actually works. 12s downtime was fine until now and to my knowledge nobody complained, because it rarely seems to happen in practice. 

> Other chains like Solana change the leader aka block author every 1.6 seconds :)

Do they have any censorship resistance? In theory it might work for them, but their architecture is fundamentally different. E.g. they don't have collators and for validators we do have byzantine assumptions and high economic guarantees.

Reply:
> For testnets we have SLOT_DURATION set to 6s.

Yes this was necessary until recently. @skunert improved the collator to be able to produce multiple blocks per slot so now we don't have to reduce slot time to have more frequent blocks anymore.

Reply:
Also if potential longer downtime is the only issue: Any ideas we have right now (including Web3) to restore censorship resistance involves accepting longer worst case downtimes.

Apart from censorship resistance longer slot times also help with maximizing throughput of elastic scaling.

TL;DR: I don't see any real advantages with going for 6s, while I do foresee at least potential issues with doing so.

Reply:
Ok. I am fine with 6s too. Mostly because for censorship, I agree we very likely will need another solution for censorship anyway and for the perf concerns, Mythical is already at 6s ... so if 12s would turn out beneficial for that reason, we would need to find a way to fix that there likely earlier anyway.